lib/version_utils.pm

```diff
@@ -249,6 +251,14 @@ sub is_rt {
     return check_var('SLE_PRODUCT', 'rt');
 }
 
+sub is_hpc {
+    return check_var('SLE_PRODUCT', 'hpc');
+}
+
+sub is_sled {
```

Comment:
That has a big overlap with `is_desktop` from lib/main_common, better use that or combine it differently

Reply:
Didn't think to look for such a method outside or `version_utils`. Will remove `is_sled` and `is_desktop` instead. Thanks.